contracts/ResilientOracle.sol

```diff
@@ -88,7 +89,8 @@ contract ResilientOracle is OwnableUpgradeable, PausableUpgradeable, ResilientOr
      * @param vToken vtoken address
      */
     function getTokenConfig(address vToken) external view returns (TokenConfig memory) {
-        return tokenConfigs[vToken];
```

Comment:
nit: If we include append `Address` to address variables as convention it is clearer and frees up the variable name for instantiating the contract e.g. `VToken vToken = VToken(vTokenAddress`

Reply:
in context of solidity I don't thing this is a practice for the reason that solidity has a dedicated type of variable `address`

Reply:
Here's a link to a conversation in slack about this- it solves more than just clarifying the variable type, it also helps resolve naming conflicts https://venusprotocol.slack.com/archives/C033EC8GCLC/p1666958405096729?thread_ts=1666916942.655199&cid=C033EC8GCLC